lang/1036.json

```diff
@@ -196,5 +196,10 @@
         "apply": "Appliquer",
         "maximized": "Agrandir",
         "no_magnet_links_found": "Pas de lien(s) magnétique(s) trouvé(s)"
```

Comment:
This line is missing a comma (,) at the end in order for the file to be valid JSON.